src/jdk.dynalink/share/classes/jdk/dynalink/linker/GuardingTypeConverterFactory.java

```diff
@@ -118,8 +118,7 @@ public interface GuardingTypeConverterFactory {
      * interface for interoperability). Invoking the {@link Supplier#get()}
      * method on the passed supplier will be subject to the same security checks
      * as {@link SecureLookupSupplier#getLookup()}. An implementation should avoid
-     * retrieving the lookup if it is not needed so as to avoid the expense of
-     * {@code AccessController.doPrivileged} call.
+     * retrieving the lookup if it is not needed.
```

Comment:
This whole passage starting with "Invoking the…" and ending with "… is not needed" should be removed now.